packages/react/src/runtime/__tests__/style.test.tsx

```diff
@@ -66,7 +66,7 @@ describe('<Style />', () => {
     });
   });
 
-  it('should warn in dev when using a dangerous pseduo selector', () => {
+  it('should warn in dev when using a dangerous pseudo selector', () => {
```

Comment:
I never get this word right myself—either way I spell it, both somehow fail spellcheck 🤣